Lib/cgi.py

```diff
@@ -13,6 +13,11 @@
 
 This module defines a number of utilities for use by CGI scripts
 written in Python.
+
+The maxlen variable can be set to an integer indicating the maximum size of a
```

Comment:
Change "The maxlen variable" to "The global variable maxlen".

Reply:
Done!

I have made the requested changes; please review again